### ISSUE TITLE
Make it possible to create an external channel to another platform

### ISSUE THREAD

**Issue by pvigoseidor** [globaleaks/GlobaLeaks#3410]
### Proposal

As an administrator, I want to be able to create an external channel that provides whistleblowers with links to relevant organizations such as anti-fraud agencies or complaint and advice services. 

### Motivation and context

Providing whistleblowers with access to additional resources and support beyond our own channels, satisfying Spanish law L 2/2023.

In addition, many of the cases we receive are often tips or suggestions rather than cases that require reporting on other platforms or websites.

**Comment by evilaliv3** [comment]
Than you @pvigoseidor 

I know that some users considers this as a valuable feature and on the contrary that others prefer instead that these aspects are handled inside the whistleblowing policy and presented via an initial site in advance of the platform. These users in fact considers that the platform that the platform should offer very simple functionalities to not confuse users, and on the contrary the organization should define good policies and specific sites to promote and clarify the possibilities.

@giorgiofraschini @javialama, what do you think?

**Comment by JaviAlama** [comment]
From our point of view and based on experience, it is true that many complaints are received and we are not competent to deal with them. 

In these cases, they are either inadmissible or closed and the reason for inadmissibility is indicated. 

I believe that creating a number of links within the project itself could reduce the simplicity and facility of the process of making a complaint.

Perhaps it would be more interesting to provide this information before linking to the mailbox, on a previous web page of the organisation.

Another option could be to create a box within the form where the whistleblower could choose the type of whistleblowing. If the typology does not exist, it would be possible to indicate that they may have chosen the wrong mailbox.

OLAF [https://fns.olaf.europa.eu/cgi-bin/disclaimer_cgi](https://fns.olaf.europa.eu/cgi-bin/disclaimer_cgi) in its whistleblowing form has a multiple checkbox where the whistleblower has to choose the reason for his/her whistleblowing.

In summary, although it is true that some complaints are received that we are not competent to resolve, there are alternatives to report them without introducing links to other channels within a mailbox.

**Comment by giorgiofraschini** [comment]
I agree with @evilaliv3 and @JaviAlama. this is better to be made outside the platform, on the information page that any organisation should have. it is the same page where the whistleblower will find the WB policy, privacy police and the link to the platform.
We should keep the IT platform as a tool where WB and the selected recipient can interact.
It is also a good practice that the recipient which was selected but believes it is not the best or correct addressee should not transfer directly but it should point out the correct or recommended addressee, giving the chance to the whistleblower to go on the other organisation/institution information page, see their policy and make a more conscious report after having collected this information.

**Comment by pvigoseidor** [comment]
From our perspective, we offer a Software as a Service (SaaS) solution to various city halls and entities that may be smaller in size and lack sufficient knowledge about anti-fraud laws, resources, and staff. 
With this feature we can ensure that whistleblowers can access higher authorities if necessary, without having to rely on the entity to inform them.

**Comment by Rjaoc** [comment]
Appreciated Pablo, Giovanni, Javi and Giorgio,

Related to the spanish law 2/2023 https://www.boe.es/buscar/act.php?lang=en&id=BOE-A-2023-4513&tn=1&p=, it is necessary to provide a complete information system. Our proposal is to improve as well the information in the context selection.

Briefly, the reason is that people could present reports to the internal and external at same time. I means, to Antifraud Office and/or to WB channel.

Is well known that previously landings are necessary, we agree with that! an is broadly used by organizations. The key of the question is that improvement, with a little development, allowing this option of linking outside, all the cases are covered.

Let me please introduce in a visually way our proposal:
![image](https://user-images.githubusercontent.com/92793609/232742619-51005477-0332-4c57-a2e3-166144425741.png)

From left to rigth: Catalonian Antifraud Office, Ethical channel and WB Channel.

In order to answer legally aspects, this could resolve further situations as superior european entities like linking to https://european-union.europa.eu/contact-eu/make-complaint_es 

Pros:
- Reducing unnecessary volume of reports.
- Improving the use of the platform. Less use but more qualified.
- Widely scope of the system. Anyone could understand the use of each channel.
- Not a high investment development.

Cons:
- "External context" could be missunderstood as internal one.
- Relative risk inserting a link. Could be solved by marking option as well as "allow external contexts" in system. 

We consider that this dev could improve a lot the solution with an small investment. If not, we are highting the dependence on external solutions like info websites that are a part of the solution that we've to cover.

**Comment by j3pc0m** [comment]
Buenas @pvigoseidor soy Javier de GrupoIWI. Me gustaría poder hablar contigo ¿es posible?

**Comment by pvigoseidor** [comment]
Hola Javier, por supuesto, escribeme en pvigo@seidoropentrends.com
